libs/math/src/math/linalg/blas/gemm_nt_vector.cpp

```diff
@@ -126,6 +126,14 @@ template class Blas<float, Signature(_C <= _alpha, _A, _B, _beta, _C),
                     Computes(_C <= _alpha * _A * T(_B) + _beta * _C),
                     platform::Parallelisation::VECTORISE>;
 
+template class Blas<
+    fetch::fixed_point::FixedPoint<16, 16>, Signature(_C <= _alpha, _A, _B, _beta, _C),
+    Computes(_C <= _alpha * _A * T(_B) + _beta * _C), platform::Parallelisation::VECTORISE>;
+
+template class Blas<
```

Comment:
no dottranspose implementation for fp128?

Reply:
fp128 does not -yet- support vectorisation, so it's intentionally omitted.